plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
+            if [[ $(git tag -l | wc -l) -eq 1 ]]; then
```

Comment:
By having "Enhancements" and "Fixes" in the release notes, it easy to see which kind of fixes commit fall into. Would you please just pipe `\n## Enhancements\n\n## Fixeş\n` into the the file?

Reply:
I can do it but I don't have any way to distinguish between fixes and enhancements. Are you planning to do that manually?

Reply:
Yes, that is the idea.

Reply:
Sounds good.